src/repository/BaseEntity.ts

```diff
@@ -483,6 +483,21 @@ export class BaseEntity {
         return this.getRepository<T>().find(options)
     }
 
+    /**
+     * Finds entities that match given WHERE conditions and at a specific timestamp.
+     *
+     * @param {Object} [options] Where conditions or a timestamp
+     * @param {Date} [timestamp] A timestamp to get datasets at a specific time
+     * @returns {Object} If entity was not found in the database this method returns null.
+     */
+    static findAsOf<T extends BaseEntity>(
```

Comment:
remove this method and add FindManyOptions property instead.

Reply:
Done